bootstrap/012_task_log.sql

```diff
@@ -0,0 +1,54 @@
+
+create or replace function internal.task_run_id()
+returns text
+AS
+$$
+     SYSTEM$TASK_RUNTIME_INFO('CURRENT_TASK_GRAPH_RUN_GROUP_ID')
+$$;
+
+let task_log_exists boolean := (exists (select * from information_schema.tables where table_schema = 'INTERNAL' and table_name = 'TASK_LOG'));
```

Comment:
nit: creating the varialbe here an using it so much later in the script was slightly jarring to me

Reply:
since truncating the old tables after inserting the rows into the new task_log table, this boolean is just an optimization and not necessary for correctness. Dropped it.